README.md

```diff
@@ -1,5 +1,5 @@
 # ITC250_branching
 
-Once aupon a time...
+Once upon a time...there was a cosplayer in the Emerald City. The cosplayer was heading to the local Comic Con at the Convention Center. The cosplayer was having a difficult time deciding what to wear. The cosplayer chose...
 
```

Comment:
The cosplayer chose to dress as batman. The cosplayer arrived the Comic Con, met people who were wearing superman and wonder woman costumes, everything was good as expected. Suddenly, a middle-aged and well-dressed couple ran toward the cosplayer and shouted "Run, Bruce! Run!" Then they lied on the ground like they were dead. "Well," the cosplayer thought, "I didn't see this coming."